bootstrap/pom.xml

```diff
@@ -16,13 +16,13 @@
     <packaging>jar</packaging>
 
     <name>BungeeCord-Bootstrap</name>
-    <description>Java 1.6 loader for BungeeCord</description>
+    <description>Java 1.7 loader for BungeeCord</description>
```

Comment:
Changes in this file are unrelated as well

Reply:
Imma revert that